cypress.config.ts

```diff
@@ -0,0 +1,34 @@
+import { defineConfig } from 'cypress'
```

Comment:
This file replaces the `cypress.env.json` one?

Reply:
No, it replaces the `cypress.json` file